File: Travel.Web/Scripts/Application/Models/SearchTravel.ts
/// <reference path="../_references.ts" />
module Application
{
    export class SearchTravel {
        
        Id: number;     
        Startdate: Date;
        Starttime: Date;
        Enddate: Date;
        StartAddress: string;
        Startpoint: MapPoint;
        EndAddress: string;
        Endpoint: MapPoint;
        Capacity: number;
        Direction: DirectionType;
        Luggage: LuggageType;
        Wait: WaitType;
        Early: EarlyType;
    }
    export enum WaitType {
        Default = 0, Five = 1, Ten = 2, Fifteen = 3, Twenty = 4
    }
    export enum EarlyType {
        Default = 0, Five = 1, Ten = 2, Fifteen = 3, Twenty = 4
    }
} 

File: Travel.Data/SearchTravel.cs
using System;
using System.Collections.Generic;
using Travel.Common.Enums;

namespace Travel.Data
{
	public class SearchTravel
	{
		public string Saddress { get; set; }
		public double? Slat { get; set; }
		public double? Slon { get; set; }
		public DateTimeOffset? Sdate { get; set; }
		public string Eaddress { get; set; }
		public double? Elat { get; set; }
		public double? Elon { get; set; }
		public DateTimeOffset? Edate { get; set; }

		public DirectionType? Direction { get; set; }
		public LuggageType? Luggage { get; set; }
		public WaitType? Wait { get; set; }
		public EarlyType? Early { get; set; }
		public int? Capacity { get; set; }
	}
}
